lib/files/schema/simulationparameter.json

```diff
@@ -0,0 +1,313 @@
+{
+    "title": "SimulationParameter",
+    "description": "It is the complete set of EnergyPlus Simulation Settings.",
+    "type": "object",
+    "properties": {
+      "type": {
+        "title": "Type",
+        "enum": [
+          "SimulationParameter"
+        ]
+      },
+      "output": {
+        "$ref": "#/definitions/SimulationOutput"
+      },
+      "run_period": {
+        "$ref": "#/definitions/RunPeriod"
+      },
+      "timestep": {
+        "title": "Timestep",
+        "default": 6,
+        "minimum": 1,
+        "type": "integer"
+      },
+      "simulation_control": {
+        "$ref": "#/definitions/SimulationControl"
+      },
+      "shadow_calculation": {
+        "$ref": "#/definitions/ShadowCalculation"
+      },
+      "sizing_parameter": {
+        "$ref": "#/definitions/SizingParameter"
+      }
+    },
+    "required": [
+      "type"
+    ],
+    "definitions": {
+      "SimulationOutput": {
+        "title": "SimulationOutput",
+        "description": "Lists the desired outputs from the simulation and the format in which to report them.",
+        "type": "object",
+        "properties": {
+          "type": {
+            "title": "Type",
+            "enum": [
+              "SimulationOutput"
+            ]
+          },
+          "reporting_frequency": {
+            "title": "Reporting_Frequency",
+            "default": "Hourly",
+            "enum": [
+              "Timestep",
+              "Hourly",
+              "Daily",
+              "Monthly",
+              "Annual"
+            ],
+            "type": "string"
+          },
+          "include_sqlite": {
+            "title": "Include_Sqlite",
+            "default": true,
+            "type": "boolean"
+          },
+          "include_html": {
+            "title": "Include_Html",
+            "default": false,
+            "type": "boolean"
+          },
+          "outputs": {
+            "title": "Outputs",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          },
+          "summary_reports": {
+            "title": "Summary_Reports",
+            "type": "array",
+            "items": {
+              "type": "string"
+            }
+          }
+        },
```

Comment:
Currently there are no methods to edit the summary reports in the OpenStudio SDK. There is an EnergyPlus measure to edit them.

Reply:
If that is the case, we might need to ship the SummaryReport EnergyPlus measure you mention along with this energy-model-measure and have the energy-model-measure assign the E+ measure to the OpenStudio model whenever there are summary reports so that it gets executed upon the simulation.  Alternatively, if we have any remaining IN2 hours for Task 3, this would be a really helpful thing to have in the OpenStudio SDK.

Reply:
I can ask Joe about implementing this in the OS SDK as well.